pygmt/helpers/decorators.py

```diff
@@ -262,14 +262,15 @@
         """,
     "panel": r"""
         panel : bool, int, or list
-            [*row,col*\|\ *index*].
-            Select a specific subplot panel. Only allowed when in subplot
-            mode. Use ``panel=True`` to advance to the next panel in the
-            selected order. Instead of *row,col* you may also give a scalar
-            value *index* which depends on the order you set via ``autolabel``
-            when the subplot was defined. **Note**: *row*, *col*, and *index*
-            all start at 0.
-         """,
+            Select a specific subplot panel. Only allowed when used in
+            :meth:`Figure.subplot`.
```

Comment:
Hm. With the current formulation, it sounds a bit like that `panel` is a parameter of `Figure.subplot`. Maybe just keep the "mode"? Unfortunately, I don't have a good formulation at the moment.

```suggestion
            Select a specific subplot panel. Only allowed when used in
            :meth:`Figure.subplot` mode.
```
or 
```suggestion
            Select a specific subplot panel. Only allowed within the code
            block related to :meth:`Figure.subplot` used as a context
            manager.
```